components/support/nimbus-cli/src/main.rs

```diff
@@ -445,12 +460,13 @@ impl CliCommand {
     fn should_kill(&self) -> bool {
         if let Some(open) = self.open_args() {
             let using_links = open.pbcopy || open.pbpaste;
+            let output_to_file = open.output.is_some();
             let no_clobber = if let Self::Open { no_clobber, .. } = self {
                 *no_clobber
             } else {
                 false
             };
-            !using_links && !no_clobber
+            !using_links && !no_clobber && !output_to_file
```

Comment:
Nit: Okay so this is just a personal suggestion for readability, I find chains like this easier to understand when combining bools in an `or` then negating it rather than `not-and`ing them. Totally a personal thing though and def not required
```suggestion
            !(using_links || no_clobber || output_to_file)
```